src/application/services/useNoteSettings.ts

```diff
@@ -20,6 +20,11 @@ interface UseNoteSettingsComposableState {
    */
   parentNote: Ref<Note | undefined>;
 
+  /**
+   * Title of the note
+   */
+  noteTitle: Ref<string>;
+
```

Comment:
note can be untitled (e.g. note contains only image), we need default value like `Untitled`